Team-C/test_wizard/lib/screens/login_screen.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:test_wizard/services/auth_service.dart';
+
+class LoginScreen extends StatefulWidget {
+  const LoginScreen ({super.key});
+  @override
+  _LoginScreenState createState() => _LoginScreenState();
+}
+
+class _LoginScreenState extends State<LoginScreen> {
+  final AuthService _authService = AuthService();
+  final TextEditingController _usernameController = TextEditingController();
+  final TextEditingController _passwordController = TextEditingController();
+  bool _isLoading = false;
+
+  void _login() async {
+    setState(() {
+      _isLoading = true;
+    });
+
+    try {
+      final token = await _authService.authenticate(
+        _usernameController.text,
+        _passwordController.text,
+      );
+
+      if (token != null) {
+        Navigator.pushReplacementNamed(context, '/home');
+      }
```

Comment:
Same thing for all of this stuff, we need to make the blue squiggly lines go away. My analyzer is giving decent errors, let me know if you can't figure it out and we can work together.